data/data/bootstrap/systemd/units/update-trust.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update Node CA Trust
+Before=kubelet.service
+
+[Service]
+ExecStart=/usr/local/bin/update-trust.sh 
```

Comment:
maybe this systemd service is not needed if coreos#120 merges?

Reply:
@patrickdillon nope you're right I don't think we would need it.  Maybe it will be a timing issue when it makes it into RHCOS?